src/compiler/scala/tools/nsc/backend/ScalaPrimitivesOps.scala

```diff
@@ -0,0 +1,238 @@
+/* NSC -- new Scala compiler
+ * Copyright 2005-2013 LAMP/EPFL
+ * @author  Martin Odersky
+ */
+
+package scala
+package tools.nsc
+package backend
+
+object ScalaPrimitivesOps extends ScalaPrimitivesOps
```

Comment:
If you'll switch names between object that are currently named `ScalaPrimitives` and `ScalaPrimitivesOps` most of the need for api changes will be gone. While this messes up the name, I think it's worth it both for API compatibility and for binary compatibility.